smalltalksrc/Slang-Tests/CSlangPrettyPrinterTest.class.st

```diff
@@ -207,6 +218,17 @@ CSlangPrettyPrinterTest >> testVisitDeclarator [
 	self assert: print equals: '*toto'
 
 
+]
+
+{ #category : #'tests-visitor' }
+CSlangPrettyPrinterTest >> testVisitDecrement [
+
+	| print |
+	print := self prettyPrint: (CDecrementNode object: (CIdentifierNode name: 'i')).
```

Comment:
I ask myself if we should not add the "Post" suffix to Decrement and Increment. Because we have "PreIncrement", "PreDecrement", but then "Increment" and "Decrement". For consistency, why not "PostDecrement"...?

What do you think?